package.json

```diff
@@ -1,71 +1,31 @@
 {
-  "name": "groqd",
-  "version": "0.13.0",
-  "license": "MIT",
-  "author": {
-    "name": "Formidable",
-    "url": "https://formidable.com"
+  "name": "groqd-workspace",
+  "private": "true",
+  "workspaces": {
+    "packages": [
+      "packages/*"
+    ]
   },
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/formidablelabs/groqd.git"
-  },
-  "homepage": "https://github.com/formidablelabs/groqd",
-  "keywords": [
-    "sanity",
-    "groq",
-    "query",
-    "typescript"
-  ],
-  "main": "dist/index.js",
-  "module": "dist/index.mjs",
-  "types": "dist/index.d.ts",
-  "exports": {
-    ".": [
-      {
-        "import": "./dist/index.mjs",
-        "types": "./dist/index.d.ts",
-        "default": "./dist/index.js"
-      },
-      "./dist/index.js"
-    ],
-    "./package.json": "./package.json"
-  },
-  "files": [
-    "dist"
-  ],
   "scripts": {
-    "test:watch": "vitest",
-    "test": "vitest run",
+    "lint": "eslint packages/**/src --quiet",
+    "test": "yarn workspaces run test",
+    "test:watch": "yarn workspaces run test:watch",
     "typecheck": "tsc --noEmit",
-    "lint": "eslint src --quiet",
     "check:ci": "yarn typecheck && yarn lint && yarn test",
-    "build": "tsup",
-    "preversion": "yarn check:ci",
+    "build": "yarn workspaces run build",
     "prepare": "yarn build",
```

Comment:
`prepare` will build the project on `yarn install` as well. Not sure if this is desired, but I confirmed that behavior locally. Pretty fast using `tsup` so I'm not sure it's harmful.

Reply:
I migrated that over from what was there. I'm not 100% sure why I have the prepare script there, I'll see if it's necessary or not. (I think with changesets, it's most likely not needed.)

Reply:
When I mirrored `groqd`'s changeset config for `react-live` I noticed it since building a FE library took longer and dropped the task as I didn't see any impact to changesets working.